src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -102,7 +107,7 @@ public Task<ResponseResult<Transaction>> FinishTransactionAsync(AccessToken toke
         {
             return Client.ProcessRequestAsync<Dto.FinishTransactionRequest, Dto.TransactionResponse, Transaction>(
                 method: HttpMethod.Put,
-                endpoint: $"tss/{tssId}/tx/{transactionId}?last_revision={lastRevision}",
+                endpoint: $"tss/{tssId}/tx/{transactionId}?tx_revision={lastRevision}",
```

Comment:
So now we're passing new revision instead of previous revision? If so, we should also rename `lastRevision` parameter.